docs/guides/developer/docs/development-process.md

```diff
@@ -86,6 +86,21 @@ modules/admin-ui/src/main/java | In case the interface of the Admin UI facade ch
 While a committer may accept a patch even if it does not meet these expectations, it is encouraged that anyone filing
 a pull request ensures that they meet these expectations.
 
+#### Merging Pull Requests
+
+After a pull request has received at least one approving review and passes the automated tests, it is ready for merging.
+Only a committer can perform a merge, so if a reviewed pull request has not yet received attention from a committer
+feel free to contact one.
+
+There are a couple of rules that committers must follow when merging pull requests. These are:
+* You cannot merge your own pull requests.
```

Comment:
This rule has been dropped. It's discouraged, but as long as it's been talked about at the technical meeting then it's free game to merge.